File: PRAKTIKUM 1/PRAKTIKUM 1/PYTHON/PRAK104-2210817110002-MUHAMMAD PASHA NABEEL.py
from cgi import print_arguments


A = 13
B = 21
a = 400000
b = 350000
Sa = a-A/100*a
sa = round(Sa) 
Sb = b-B/100*b
sb = round(Sb)

print("Harga sepatu A adalah ",a)
print("Harga sepatu B adalag ",b)
print("Sepatu A mendapat diskon 13% sehingga harganya menjadi ", sa)
print("Sepatu B mendapat diskon 21% segingga harganya menjadi ", sb)

File: Modul 1/Soal 4/PRAK104-2210817110002-MUHAMMAD PASHA NABEEL.c
#include<stdio.h>

int main()
{
    float A,B,a,b,hasil1,hasil2;

    A = 13;
    B = 21;
    a = 400000;
    b = 350000;

    printf("Harga sepatu A adalah %.0f\n", a);
    printf("Harga sepatu B adalah %.0f\n", b);

    hasil1 = a-A/100*a;
    hasil2 = b-B/100*b;

    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi %.0f\n", hasil1);
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi %.0f\n", hasil2);

    return 0;
}
